### PR TITLE
feat: less Mattermost events

### PR DESCRIPTION
**Pull request by CHHC-L**
- In `.env.example`, added Mattermost team and token setting to run `create-channels-on-mm` more easily.
- In `./joint_teapot/workers/mattermost.py`, trigger messages on less events.
- Updated related part of README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ close all issues and pull requests in gitea organization
 
 create channels for student groups according to group information on gitea. Optionally specify a prefix to ignore all repos whose names do not start with it. Optionally specify a suffix to add to all channels created.
 
-Example: `python3 -m joint_teapot create_channels_for_groups --prefix p1 -suffix -private` will fetch all repos whose names start with `"p1"` and create channels on mm for these repos like "p1team1-private". Members of a repo will be added to the corresponding channel.
+Example: `python3 -m joint_teapot create-channels-on-mm --prefix p1 --suffix -private --invite-teaching-team` will fetch all repos whose names start with `"p1"` and create channels on mm for these repos like "p1team1-private". Members of a repo will be added to the corresponding channel. And teaching team (maybe adjust `mattermost_teaching_team` list in `./joint_teapot/config.py`) will be invited to the channels.
```
Comment (BoYanZh) [inline README.md:63]: The value in the list can be set with `.env` file.

```
MATTERMOST_TEACHING_TEAM=["A", "B", "C"]
```

**Review by BoYanZh — CHANGES_REQUESTED** [review]

**Comment by wznmickey** [comment]
Where can we get the mattermost token?

**Comment by CHHC-L** [comment]
Changed `.env.example`.

**Comment by CHHC-L** [comment]
> Where can we get the mattermost token?

Channel admin should allows a user generate token in `System Console -> User Management,` then the user can get token from his `Profire -> Security -> Personal Access Tokens`. Refer to [this page](https://docs.mattermost.com/developer/personal-access-tokens.html) .

**Review by BoYanZh — APPROVED** [review]

**Pull request merged**